java/src/main/java/ai/rapids/cudf/ColumnFilterOptions.java

```diff
@@ -21,7 +21,6 @@
 import java.util.ArrayList;
 import java.util.Collection;
 import java.util.List;
-import java.util.function.Supplier;
```

Comment:
This was unrelated to this PR and removes an unused import. We can revert this change if needed.